test/e2e/utils/test_context.go

```diff
@@ -66,14 +67,15 @@ func NewTestContext(binaryName string, env ...string) (*TestContext, error) {
 	}
 
 	return &TestContext{
-		TestSuffix: testSuffix,
-		Domain:     "example.com" + testSuffix,
-		Group:      testGroup,
-		Version:    "v1alpha1",
-		Kind:       "Foo" + testSuffix,
-		Resources:  "foo" + testSuffix + "s",
-		ImageName:  "e2e-test/controller-manager:" + testSuffix,
-		CmdContext: cc,
+		TestSuffix:    testSuffix,
+		Domain:        "example.com" + testSuffix,
+		Group:         testGroup,
+		Version:       "v1alpha1",
+		SecondVersion: "v1",
+		Kind:          "Foo" + testSuffix,
+		Resources:     "foo" + testSuffix + "s",
+		ImageName:     "e2e-test/controller-manager:" + testSuffix,
+		CmdContext:    cc,
```

Comment:
Could we use the fmt.Sprintf to concat the values above?

Reply:
Done.